src/moonlink/src/storage/wal.rs

```diff
@@ -8,11 +8,35 @@ use crate::table_notify::TableEvent;
 use crate::Result;
 use futures::stream::{self, Stream};
 use futures::{future, StreamExt};
-use more_asserts::assert_ge;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
+use std::path::Path;
 use std::pin::Pin;
 use std::sync::Arc;
+use tokio::sync::mpsc::Sender;
+
+pub const DEFAULT_WAL_FOLDER: &str = "wal";
+
+#[derive(Debug, Clone)]
+pub struct WalConfig {
+    accessor_config: AccessorConfig,
+}
+
+impl WalConfig {
+    pub fn default_wal_config_local(table_id: u32, base_path: &Path) -> WalConfig {
+        let wal_storage_config = StorageConfig::FileSystem {
+            root_directory: base_path
+                .join(DEFAULT_WAL_FOLDER)
+                .join(table_id.to_string())
```

Comment:
need to add database id as well

Reply:
example for iceberg: https://github.com/Mooncake-Labs/moonlink/blob/32b76bfd4a6e0b64e93af756c959e1b5bc36d694/src/moonlink_connectors/src/pg_replicate/table_init.rs#L67-L71